src/add_to_mean.d.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * When adding a new value to a list, one does not have to necessary
+ * recompute the mean of the list in linear time. They can instead use
+ * this function to compute the new mean by providing the current mean,
+ * the number of elements in the list that produced it and the new
+ * value to add.
+ *
+ * @since 2.5.0
+ * @param {number} mean current mean
+ * @param {number} n number of items in the list
+ * @param {number} newValue the added value
+ * @returns {number} the new mean
+ *
+ * @example
+ * addToMean(14, 5, 53); // => 20.5
+ */
+declare function addToMean(mean: number, n: number, newValue: number): number
```

Comment:
Do typescript definitions have to repeat the documentation strings?

Reply:
🤔 Well in practice no... but then you'd lose the JSDocs information provided such as the `@since`, `@example` & the description.

## With JSDocs

![image](https://user-images.githubusercontent.com/550895/31589713-4ed7ddaa-b1d4-11e7-8502-48aabbb135e6.png)

## Without JSDocs

![image](https://user-images.githubusercontent.com/550895/31589717-5e85adb8-b1d4-11e7-9b4b-d4e8446af15a.png)